### PR TITLE
xonsh AppImage to xonsh GitHub assets

### PR DESCRIPTION
**Pull request by anki-code**
Required Rever AppImage activity - https://github.com/regro/rever/pull/215

### CODE REVIEW

**Comment by gforsyth** [comment]
Note -- this relies on regro/rever#215 being merged in.

**Comment by anki-code** [comment]
> Note -- this relies on regro/rever#215 being merged in.

regro/rever#215 is merged.

File: rever.xsh [hunk 1]
```diff
@@ -35,3 +35,5 @@ $DOCKER_INSTALL_COMMAND = ('rm -rf .cache/ __pycache__/ */__pycache__ */*/__pyca
                            './setup.py install')
 $DOCKER_GIT_NAME = 'xonsh'
 $DOCKER_GIT_EMAIL = 'xonsh@googlegroups.com'
+
+$GHRELEASE_ASSETS = ['xonsh-x86_64.AppImage']
```
Comment (scopatz) [inline rever.xsh:39]: I think this needs to include the default assets

Comment (scopatz) [reply rever.xsh:39]: i.e.

```py
from rever.activities.ghrelease import git_archive_asset

$GHRELEASE_ASSETS = [git_archive_asset(), 'xonsh-x86_64.AppImage']
```

Comment (anki-code) [reply rever.xsh:39]: @scopatz This PR allows to change the code. Please make changes as you see this. 

I also can just put the lines from your example to the branch.

Comment (scopatz) [reply rever.xsh:39]: Ahh yeah... I have no idea how to check if a PR allows edits

Comment (scopatz) [reply rever.xsh:39]:
```$ git push --set-upstream xxh rever_appimage
ERROR: Permission to xxh/xonsh.git denied to scopatz.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

It failed to let me push

Comment (anki-code) [reply rever.xsh:39]: Hmm, strange. Ok, I've added you as writer to xxh/xonsh. Please try again.

Comment (scopatz) [reply rever.xsh]: Thanks! I have just pushed up some changes

**Comment by anki-code** [comment]
@scopatz @gforsyth This looks like ready to merge )

**Comment by scopatz** [comment]
Thanks!

**Comment by anki-code** [comment]
Yooooooooooooooohoooooooooooooooooo! I see `xonsh.AppImage` in the 0.9.20 assets! Thank you @scopatz!

**Comment by scopatz** [comment]
Yep! It made it!

**Pull request merged**